df.viewscreen.xml

```diff
@@ -2208,7 +2216,7 @@
         </stl-vector>
         <int32_t name='background_idx'/>
 
-        <stl-vector name='unk_35_a' type-name='int32_t'/> castes?
+        <stl-vector name='allowed_castes' type-name='int32_t'/>
```

Comment:
Is there a possible ref-target attribute here?

Reply:
Added, ideally it should also have refers-to to be useful, but I don't know refers-to syntax well enough.